manifests/0000_70_cluster-network-operator_03_daemonset.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
   namespace: openshift-network-operator
   labels:
     name: network-operator
+  annotations:
+    config.openshift.io/inject-proxy: operator
```

Comment:
looks like the container name is network-operator, not operator, in this deployment.

Reply:
Updated to container name `network-operator` in latest commit.